### PR TITLE
Fix UNIMPLEMENTED handler and double event call

### LINKED ISSUE #19: Questionable SSH_MSG_UNIMPLEMENTED handling
The SSH_MSG_UNIMPLEMENTED handler currently sends unimplemented message back:

https://github.com/Aimeast/FxSsh/blob/dbacd36d14698d0f85a28c44301e99e172e64113/FxSsh/Session.cs#L600-L602

It's weird behavior. `openssh`, for example, just logs this message without doing anything with it:

https://github.com/openssh/openssh-portable/blob/62dd70613b77b229f53db3cc1c3e8a206fa2b582/packet.c#L1725

### PR DESCRIPTION
**Pull request by DCNick3**
fix #19 #20

### CODE REVIEW

**Comment by Aimeast** [comment]
Thank you! looks good. I will review again when I come back to computer.

File: FxSsh/Session.cs [hunk 1]
```diff
@@ -131,7 +131,10 @@ internal void EstablishConnection()
                 while (_socket != null && _socket.Connected)
                 {
                     var message = ReceiveMessage();
-                    HandleMessageCore(message);
+                    if (message is UnknownMessage)
+                        SendMessage((message as UnknownMessage).MakeUnimplementedMessage());
+                    else
+                        HandleMessageCore(message);
```
Comment (Aimeast) [inline FxSsh/Session.cs:137]: Please use new syntax
```cs
    var message = ReceiveMessage();
    if (message is UnknownMessage unknownMessage)
        SendMessage(unknownMessage.MakeUnimplementedMessage());
    else
        HandleMessageCore(message);
```

File: FxSsh/FxSsh.csproj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (Aimeast) [inline FxSsh/FxSsh.csproj]: please remove unnecessary changes

File: FxSsh/Messages/UnimplementedMessage.cs [hunk 3]
```diff
@@ -1,22 +1,22 @@
-﻿using System;
-
-namespace FxSsh.Messages
+﻿namespace FxSsh.Messages
```
Comment (Aimeast) [inline FxSsh/Messages/UnimplementedMessage.cs]: please remove unnecessary changes

File: FxSsh/Messages/UnimplementedMessage.cs [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
 {
     [Message("SSH_MSG_UNIMPLEMENTED", MessageNumber)]
     public class UnimplementedMessage : Message
     {
         private const byte MessageNumber = 3;
-
+
```
Comment (Aimeast) [inline FxSsh/Messages/UnimplementedMessage.cs]: please remove unnecessary changes

File: FxSsh/Messages/UnimplementedMessage.cs [hunk 5]
```diff
@@ -1,22 +1,22 @@
...
+        }
+
         protected override void OnGetPacket(SshDataWorker writer)
         {
-            writer.Write(MessageNumber);
             writer.Write(SequenceNumber);
         }
     }
-}
+}
```
Comment (Aimeast) [inline FxSsh/Messages/UnimplementedMessage.cs]: with an EOL(end of line) please

**Comment by Aimeast** [comment]
It's almost finished, could you please make some changes?

**Comment by DCNick3** [comment]
Which? I updated and squashed it to one commit recently..

**Comment by Aimeast** [comment]
Yesterday, I have commented on code review page, just a little detailed commented.

**Comment by DCNick3** [comment]
Weird, can't see any of the reviews..

**Comment by Aimeast** [comment]
It's my fault, I have not submit the review. Now you may will see.

**Comment by DCNick3** [comment]
Updated

**Pull request merged**